app/consolidator/scheduler/JobSchedulerTask.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package consolidator.scheduler
+
+import akka.actor.ActorSystem
+import consolidator.FormConsolidatorActor
+import javax.inject.Inject
+import org.slf4j.{ Logger, LoggerFactory }
+import play.api.inject.ApplicationLifecycle
+
+import scala.concurrent.Future
+
```

Comment:
Creates actor system and schedules the jobs. Also hooks the scheduler to the application lifecycle, to ensure a clean shutdown on exit